vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/style/Text.java

```diff
@@ -0,0 +1,413 @@
+package com.vaadin.flow.component.map.configuration.style;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import com.vaadin.flow.component.map.configuration.AbstractConfigurationObject;
+import com.vaadin.flow.component.map.configuration.Constants;
+
+import java.io.Serializable;
+
+/**
+ * Text style that defines how to render texts, such as labels, on the map.
+ */
+public class Text extends AbstractConfigurationObject {
```

Comment:
Not sure about the naming. For one this conflicts with the existing Flow `Text` class, and then the name is misleading since this class (in contrast to the OpenLayers class) doesn't contain actual text data, only styles. Maybe rename it `TextStyle`?

Reply:
Yeah, even for the OpenLayers class I find it a bit confusing that you `style.getText().setText()` to actually set the text content. `TextStyle` seems to be more precise with the purpose of this class.

Reply:
Renamed the class to `TextStyle`. I also renamed the property in `Style` to `textStyle`, and added a Jackson annotation to map it to `text` on serialization.